autoload/lightline/gitdiff/algorithms/word_diff_porcelain.vim

```diff
@@ -17,16 +17,17 @@ function! lightline#gitdiff#algorithms#word_diff_porcelain#calculate(buffer) abo
   let l:lines_modified = len(filter(copy(l:changes), { idx, val -> val ==# 'M' }))
 
   let l:ret = {}
+  let l:show_empty_indicators = exists('g:lightline#gitdiff#show_empty_indicators') && g:lightline#gitdiff#show_empty_indicators
```

Comment:
You could use the following, which does checking for existence of the key. If the key doesn't exist, zero is returned, see `:h get()`.

```suggestion
  let l:show_empty_indicators = get('g:lightline#gitdiff#show_empty_indicators')
```